client/src/redux/user/user.actions.ts

```diff
@@ -1,14 +1,27 @@
-import { User, UserActionTypes, FETCH_CURRENT_USER, SET_CURRENT_USER } from './user.types';
+import {
+	User,
+	UserActionTypes,
+	FETCH_CURRENT_USER,
+	SET_CURRENT_USER,
+	ADD_PROJECT_TO_USER,
+} from './user.types';
 
 export function fetchCurrentUser(): UserActionTypes {
 	return {
 		type: FETCH_CURRENT_USER,
 	};
 }
 
-export function setCurrentUser(user: User) {
+export function setCurrentUser(user: User): UserActionTypes {
 	return {
 		type: SET_CURRENT_USER,
 		payload: user,
 	};
 }
+
+export function addProjectToUser(projectId: string): UserActionTypes {
+	return {
+		type: ADD_PROJECT_TO_USER,
+		payload: projectId,
+	};
+}
```

Comment:
As I recall, I was trying to accomplish a more generalized form of this action. Something more like "modify user action" would be better but as of right now this is okay, add a note above this for later